noWord/common/NWGenerator.py

```diff
@@ -100,6 +109,12 @@ def setDecorations(self, obj):
             for decoration in decorations:
                 self.doc.addDecoration(getattr(obj, decoration))
 
+    def setTransformations(self, obj):
```

Comment:
Is this used anywhere ?

Reply:
Yes. when you add custom transfos in the sub project (sub module). like decorations.